interface/handler/blackjack.go

```diff
@@ -0,0 +1,99 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/JosueSdev/golang-bootcamp-2020/interface/service"
+	"github.com/JosueSdev/golang-bootcamp-2020/usecase/blackjack"
+)
+
+type blackjackHandler struct {
+	Service service.Deck
+}
+
+//Blackjack provides http handlers for the blackjack game
+type Blackjack interface {
+	PutTable(w http.ResponseWriter, r *http.Request)
+	GetGame(w http.ResponseWriter, r *http.Request)
+}
+
+//NewBlackjackHandler constructs a blackjack handler eith dependency injection
+func NewBlackjackHandler(s service.Deck) Blackjack {
+	return &blackjackHandler{s}
+}
+
+//PutTable fetches a new deck an uses it to write/replace the csv datastore
+func (bj *blackjackHandler) PutTable(w http.ResponseWriter, r *http.Request) {
+	if err := bj.Service.ReloadDeck(); err != nil {
+		HTTPProblem(w, 500)
```

Comment:
If you already are importing `"net/http"`, why not to use the status code constants instead of the magic numbers `500`?